compiler-rt/lib/asan/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ set(ASAN_CFLAGS ${SANITIZER_COMMON_CFLAGS})
 
 append_list_if(MSVC /Zl ASAN_CFLAGS)
 
-set(ASAN_COMMON_DEFINITIONS ${COMPILER_RT_ASAN_SHADOW_SCALE_DEFINITION})
+set(ASAN_COMMON_DEFINITIONS)
```

Comment:
Does `unset` work?

If this is removed, will the build break?

Reply:
Thanks for pointing this out!
I tested both locally and in buildbot https://lab.llvm.org/buildbot/#/builders/37/builds/34788
Do you think `set(ASAN_COMMON_DEFINITIONS "")` is better?

Reply:
Yes, `""` might be better

Reply:
done